apps/ui-layout/registry/components/button/btn-click-down.tsx

```
@@ -3,7 +3,7 @@
 const ButtonClickDown = () => {
   return (
     <>
-      <button className='group relative inline-flex h-12 items-center justify-center overflow-hidden rounded-md   bg-linear-to-r dark:from-[#070e41] dark:to-[#263381] from-[#f6f7ff] to-[#f5f6ff] dark:border-[rgb(76_100_255)] border-2 border-[#263381] bg-transparent px-6 font-medium dark:text-white text-black transition-all duration-100 dark:[box-shadow:5px_5px_rgb(76_100_255)] [box-shadow:5px_5px_rgb(38_51_129)] active:translate-x-[3px] active:translate-y-[3px] active:[box-shadow:0px_0px_rgb(38_51_129)] dark:active:[box-shadow:0px_0px_rgb(76_100_255)]'>
+      <button className='group relative inline-flex h-12 items-center justify-center overflow-hidden rounded-md   bg-linear-to-r dark:from-[#070e41] dark:to-[#263381] from-[#f6f7ff] to-[#f5f6ff] dark:border-[rgb(76_100_255)] border-2 border-[#263381] bg-transparent px-6 font-medium text-foreground transition-all duration-100 dark:[box-shadow:5px_5px_rgb(76_100_255)] [box-shadow:5px_5px_rgb(38_51_129)] active:translate-x-[3px] active:translate-y-[3px] active:[box-shadow:0px_0px_rgb(38_51_129)] dark:active:[box-shadow:0px_0px_rgb(76_100_255)]'>
```

Biome (2.4.16)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)